src/plugins/next-image/plugin.ts

```diff
@@ -42,6 +57,12 @@ export function vitePluginNextImage(
         return null;
       }
 
+      // For SVG files, only process if they don't have ?react parameter and SVG processing is enabled
+      const isSvg = /\.svg$/.test(source);
+      if (isSvg && (excludeSvg || queryA === "react")) {
```

Comment:
Could we extract the `svgr`'s `include`, and `exclude` options from the `config` hook and use them to determine whether the next-image plugin should proceed?